docs/dev/components/core-rest.md

```diff
@@ -0,0 +1,72 @@
+# Core REST API
+
+The [Core REST API module](https://github.com/gchq/Gaffer/tree/master/rest-api/core-rest) contains a Gaffer REST API.
+
+Gaffer Stores have modules extending the core-rest and adds in the dependency for the Gaffer Store. So if you want to use the Accumulo Store REST API, you can use the `accumulo-rest` `.war`, or `map-rest` for the Map Store.
+
+For an example of using the core REST API please see the [example/road-traffic module](https://github.com/gchq/Gaffer/tree/master/example/road-traffic).
+
+## How to modify the Core REST API for your project
+
+You can easily make changes or additions to the core REST API for your project. You will need to create a new Maven module to build your core REST API. In your POM you should configure the `maven-dependency-plugin` to download the core Gaffer REST API `.war` and extract it. When Maven builds your module it will unpack the core war, add your files and repackage the war. If you wish to override a file in the core war then you can do this by including your own file with exactly the same name and path.
```

Comment:
I know its not really been done for any of the other markdown but its usually good practise to wrap md text at a fixed length (usually 80) so you don't end up with one massive line.

Reply:
Yes it usually is, although I think this should be refactored globally and/or enforced with linting. In lots of places long lines have been used as this wraps in the IDE and in unnoticeable.

Reply:
Agreed, probably a task in its self to go back over existing files and add the line breaks at an agreed length, wondering if we want to implement this as an unwritten rule on new markdown content for now on? (not saying have to do it for these docs as I know they are kind of a lift and shift)

Reply:
I don't think we want any unwritten rules. I'd suggest we make use of a Markdown linter which runs on Python. Hopefully we'd be able to set a rule for one sentence per line unless there's space for more. I don't think we want sentences cut off and split across lines though. Also we'd want to set a sensible modern length and not 80 chars!